main.py

```diff
@@ -0,0 +1,21 @@
+import os
+import dotenv
+from telegram.ext import ApplicationBuilder
+from handlers import handlers
+
+dotenv.load_dotenv()
+bot_api_key = os.getenv("BOT_API_KEY")
+
+
+def main():
+    if bot_api_key is None:
+        raise ValueError("BOT_API_KEY environment variable not set")
+
+    application = ApplicationBuilder().token(bot_api_key).build()
+    application.add_handlers(handlers)
+    print("Starting wolt bot")
```

Comment:
Prints are usually for hacky-debugging needs (like console.log() in Javascript). Better to use the logging module, you can read about [here ](https://realpython.com/python-logging/)